test/Umbraco.Headless.Client.Net.Tests/Management/FormServiceFixture.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Refit;
+using RichardSzalay.MockHttp;
+using Umbraco.Headless.Client.Net.Configuration;
+using Umbraco.Headless.Client.Net.Management;
+using Xunit;
+
+namespace Umbraco.Headless.Client.Net.Tests.Management
+{
+    public class FormServiceFixture
```

Comment:
Missing a few test cases for when submitting an entry fails due to validation or due to some other error.

Reply:
Added a test for when an error ocours when submitting an entry